style.css

```diff
@@ -0,0 +1,81 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
+
+body {
+  background-color: #e5e5e5;
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  margin: 15%;
+}
```

Comment:
- [ ] Well done 👏🏻 you made the changes as per the previous code reviewer's request, but it seems like you didn't make all the changes because your previous code reviewer mentioned regarding your design is not look good and does not cover the mobile screen size properly. Fortunately, I have found the issue and below is the suggestion for solving the issue.

```suggestion
body {
  background-color: #e5e5e5;
}
```
### After the above changes

- [ ] Kindly use the `box-model` technique for your style because as I can see that there is some white border around your application which you can easily remove by using the CSS resetter as given below

```
*,
*::after,
*::before {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
}
```

###  Design Comparison

<table>
<tr>
<th>Your Design Before Resetter</th>
<th>Your Design After Resetter</th>
</tr>

<tr>
<td>

![image](https://github.com/medijdir/portfolio_project/assets/66476329/ed89135c-fea7-4368-acaa-5bf8bae3badb)


</td>

<td>

![Screenshot from 2023-06-07 13-48-34](https://github.com/medijdir/portfolio_project/assets/66476329/94a9eb6b-9390-4b67-94bb-320d48a3e8db)


</td>
</tr>
</table>